Allow custom repository when pushin in fork (#144)
peter-evans/create-pull-request: docs/concepts-guidelines.md

          token: ${{ secrets.GITHUB_TOKEN }}
```

> ### Push in fork

Reviewer: ```suggestion
### Push pull request branches to a fork
```

Reviewer: Also, please add a link in the contents at the top of the page.
```
  - [Push pull request branches to a fork](#push-pull-request-branches-to-a-fork)
```

Author: Thanks, link added

To enforce security, you can use a dedicated user using [machine account](https://help.github.com/en/github/site-policy/github-terms-of-service#3-account-requirements).
This user has no access to the main repository, it will use their own fork to push code and create the pull request.

1. Create a new github user, then login with this user.
2. fork the repository.
3. create a [Personal Access Token (PAT)](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line).
4. logout and go back to your main user.
5. Add a secret to the repository containing the above PAT.

> 6. As show in the example below, switch the git remote to the fork's url after checkout and set the option `request-on-parent`

Reviewer: ```suggestion
6. As shown in the example below, switch the git remote to the fork's url after checkout and set the action input `request-on-parent` to `true`.
```

> 6. As show in the example bellow, switch the git remote to the fork's url after checkout and set the option `request-on-parent`

Reviewer 2: ```suggestion
6. As show in the example below, switch the git remote to the fork's url after checkout and set the option `request-on-parent`
```
